tests/recipes/test_eleuther_eval.py

```diff
@@ -83,8 +83,8 @@ def test_eval_recipe_errors_without_lm_eval(self, caplog, monkeypatch, tmpdir):
         """.split()
 
         monkeypatch.setattr(sys, "argv", cmd)
-        with pytest.raises(SystemExit):
+        with pytest.raises(SystemExit, match="1"):
```

Comment:
I can just look this up, but what are these magic values?

Reply:
Denotes SystemExits that contain errors.